common/xml/fpga_architecture.xsd

```diff
@@ -584,6 +584,40 @@
     <xs:attribute name="name" type="xs:string" use="required"/>
   </xs:complexType>
 
+  <xs:complexType name="map">
+    <xs:attribute name="from" type="xs:string" use="required"/>
+    <xs:attribute name="to" type="xs:string" use="required"/>
+  </xs:complexType>

```

Comment:
For now, here there should be also a `num_pin` attribute which is optional. I still have to modify VtR XML parser to allow the definition of the port range (e.g. DIN1[3:0] maps to DIN[7:4] and DIN2[3:0] maps to DIN[3:0]).

Reply:
`num_pin` or `num_pins`?

Reply:
Right, `num_pins` is correct